.github/workflows/clang-asan-ubsan.yml

```diff
@@ -0,0 +1,26 @@
+name: ClangASanUBSan
+
+on: [push]
+
+jobs:
+  clang-tsan:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout Sources
+      uses: actions/checkout@v1
+
+    - name: Install Dependencies
+      run: |
+        sudo apt-get install ninja-build
+      
+    - name: Build and test
+      run: |
+        ./build-deps.sh -DCMAKE_C_COMPILER=clang -DENABLE_SANITIZERS=ON -DSANITIZERS=",address,undefined"
+        cd build
+        cmake -GNinja -DCMAKE_C_COMPILER=clang -DENABLE_SANITIZERS=ON -DSANITIZERS=",address,undefined" -DCMAKE_PREFIX_PATH=`pwd`/deps/install/ ..
+        ninja
```

Comment:
trivial: Do we need `ninja` for this? We can just let cmake use whatever build system in on the machine, and issue the build command like so:
`cmake --build .`

Reply:
We could also just omit the -GNinja and just use `cmake --build .`